### PR TITLE
Allowing 0 vesting cliff

### PR DESCRIPTION
**Pull request by Peartes**
Allow for clients to set a vesting cliff of 0 days to disburse seed tokens out immediately after max has been reached or after the end date of project launch. Addresses this [issue](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=905c5af68ab44851bc60603aa60eb678)

### CODE REVIEW

File: src/newSeed/stage4.ts [hunk 1]
```diff
@@ -90,17 +90,17 @@ export class Stage4 extends BaseStage {
...
-    } else if (!(Number(this.seedConfig.seedDetails.vestingCliff) > 0)) {
-      message = "Please enter a number greater than zero for \"with a cliff of\" ";
-    } else if (BigNumber.from(this.seedConfig.seedDetails.fundingTarget).gt(this.seedConfig.seedDetails.fundingMax)) {
-      message = "Please enter a value for Funding Target smaller than Funding Max";
+    } else if (!(this.seedConfig.seedDetails.vestingCliff >= 0)) {
+      message = "Please enter a value for \"with a cliff of\" ";
     } else if (this.seedConfig.seedDetails.vestingCliff > this.seedConfig.seedDetails.vestingDays) {
       message = "Please enter a value of \"with a cliff of\" less than \"Seed tokens vested for \"";
+    } else if (BigNumber.from(this.seedConfig.seedDetails.fundingTarget).gt(this.seedConfig.seedDetails.fundingMax)) {
+      message = "Please enter a value for Funding Target smaller than Funding Max";
```
Comment (dkent600) [inline src/newSeed/stage4.ts:103]: can you please confirm whether target and max can be equal?  I don't see why not.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/newSeed/stage4.ts (hunk 1 above)
Comment (Peartes) [reply src/newSeed/stage4.ts:103]: it can

Comment (dkent600) [reply src/newSeed/stage4.ts:103]: ok, can you please correct this validation accordingly?

Comment (Peartes) [reply src/newSeed/stage4.ts:103]: This validation would allow for the 2 values to be equal

Comment (dkent600) [reply src/newSeed/stage4.ts:103]: This:

> "Please enter a value of \"with a cliff of\" less than \"Seed tokens vested for \""

Should read:

> "Please enter a value of \"with a cliff of\" less than or equal to \"Seed tokens vested for \""

Comment (dkent600) [reply src/newSeed/stage4.ts:103]: This:

> "Please enter a value for Funding Target smaller than Funding Max"

Should read:

> "Please enter a value for Funding Target less than or equal to Funding Max"

Comment (Peartes) [reply src/newSeed/stage4.ts]: Got you, the validation messages

**Review by dkent600 — APPROVED** [review]

**Pull request merged**